src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -542,8 +542,10 @@ private void SetNodeInfo(CustomNodeInfo newInfo)
                     // it will be useful to know when. It might happen during a complex versioning publish workflow.
                     // This represents the case where a previous info was not from a package, but the current info
                     // has an owning package.
-                    var looseCustomNodeToPackageMessage = String.Format("Attempting to load a new definition for customNode {0}," +
-                        " loaded by package {1}, but A previous definition exists with no associated package, ", info.Name, newInfo.PackageInfo);
+                    var looseCustomNodeToPackageMessage = String.Format("Attempting to load customNode {0}," +
+                        " loaded by package {1}, but A previous definition named {2} exists with no associated package." +
+                        " The new customNode definition has been loaded, but Dynamo may in an unstable state, please avoid loading" +
+                        " multiple custom nodes with the id {3}. ", newInfo.Name, newInfo.PackageInfo, info.Name, info.FunctionId);
```

Comment:
I wonder if it's useful to show the ID here. What can users do with it even if they know it?

Reply:
look through their dyfs? I think this info is more for us than users.

Reply:
we could change it to say ... "please avoid loading multiple custom nodes with the same id" ?